src/openrct2/platform/windows.c

```diff
@@ -1213,7 +1213,12 @@ bool platform_setup_uri_protocol()
 						HKEY hMuiCacheKey;
 						if (RegCreateKeyW(hRootKey, MUI_CACHE, &hMuiCacheKey) == ERROR_SUCCESS) {
 							swprintf_s(buffer, sizeof(buffer), L"%s.FriendlyAppName", exePath);
+#ifdef __MINGW32__
+							// MINGW defines RegSetKeyValueW's signature incorrectly
+							RegSetKeyValueW(hMuiCacheKey, NULL, (LPCSTR)buffer, REG_SZ, (LPCSTR)L"OpenRCT2", sizeof(L"OpenRCT2") + 1);
```

Comment:
Can you please leave a note that this refers to mingw-w64 (apparently there is more than one) and that a fix was submitted upstream, so it should be removed when we can use their next release: https://sourceforge.net/p/mingw-w64/mingw-w64/ci/da9341980a4b70be3563ac09b5927539e7da21f7/